base/pkg/libgit2/error.jl

```diff
@@ -0,0 +1,107 @@
+module GitErrorConst
+    const GIT_OK          = Cint(0)
+    const ERROR           = Cint(-01)
+    const ENOTFOUND       = Cint(-03)
+    const EEXISTS         = Cint(-04)
+    const EAMBIGUOUS      = Cint(-05)
+    const EBUFS           = Cint(-06)
+    const EUSER           = Cint(-07)
+    const EBAREREPO       = Cint(-08)
+    const EUNBORNBRANCH   = Cint(-09)
+    const EUNMERGED       = Cint(-10)
+    const ENONFASTFORWARD = Cint(-11)
+    const EINVALIDSPEC    = Cint(-12)
+    const EMERGECONFLICT  = Cint(-13)
+    const ELOCKED         = Cint(-14)
+    const PASSTHROUGH     = Cint(-30)
+    const ITEROVER        = Cint(-31)
+end
+
+const git_error_code = Dict{Int,Symbol}(
```

Comment:
Here in particular, if you use an enum in the block above you should be able to do things like `GitError[-03]` and get `notfound` without defining the dict manually.